app/controllers/users_controller.rb

```diff
@@ -1097,6 +1098,19 @@ def revoke_account
     end
   end
 
+  def revoke_auth_token
+    user = fetch_user_from_params
+    guardian.ensure_can_edit!(user)
+
+    UserAuthToken.where(user_id: user.id).destroy_all
+
+    MessageBus.publish "/file-change", ["refresh"], user_ids: [user.id]
```

Comment:
Is this intended to reload the page automatically? If so, we probably want to do a redirect back to the user preferences page.

Reply:
Destroying all the `UserAuthToken` will log the user out. Not sure redirecting back to the user preferences page is useful here?